### ISSUE TITLE
Using vue devtools will error.

### ISSUE THREAD

**Issue by coolman90011** [quasarframework/quasar#16193]
### What happened?

1. npm init quasar (chose webpack cli version).
2. add script to packages.json(""electron": "npx quasar dev -m electron -p 8090 --devtools",")
![image](https://github.com/quasarframework/quasar/assets/52725981/39c8b59e-0234-4426-9488-aa43d2ad7fbf)
3.excute command "yarn electron" and will error.
![image](https://github.com/quasarframework/quasar/assets/52725981/5ac33afe-e860-417b-a1e5-e55c99d4ce9c)

### What did you expect to happen?

vue devtools will open correctly.

### Reproduction URL

https://github.com/coolman90011/quasar2webpack

### How to reproduce?

1. npm init quasar (chose webpack cli version).
2. add script to packages.json(""electron": "npx quasar dev -m electron -p 8090 --devtools"")
3. excute command "yarn electron" and will error.

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite)

### Platforms/Browsers

Electron

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19045) - win32/x64
NodeJs - 16.15.0

Global packages
  NPM - 8.5.5
  yarn - 1.22.19
  @quasar/cli - 2.2.2
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.12.4 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-webpack - 3.9.2 -- Quasar Framework App CLI with Webpack
  @quasar/extras - 1.16.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.3.4 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.2.4
  pinia - 2.1.6 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  electron - 26.0.0 -- Build cross platform desktop apps with JavaScript, HTML, and CSS
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.22.10 -- Babel compiler core.
  webpack - 5.88.2 -- Packs ECMAScript/CommonJs/AMD modules for the browser. Allows you to split your codebase into multiple bundles, which can be loaded on demand. Supports loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.11.1 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.9.4 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - DESKTOP-9ACL4HO
  乙太網路 - 192.168.1.157
```

**Comment by rstoenescu** [comment]
This is a known issue of vue devtools package.
Reference: https://github.com/vuejs/devtools/issues/1886

However, I've already worked around it for the next major versions of our CLIs (currently unreleased to the public).
I'm now in the process of backport the implementation to the current CLI versions.

**Comment by rstoenescu** [comment]
Enhancements will be available in q/app-vite 1.4.4
Fixes will be available in q/app-webpack 3.9.3
